recipe/meta.yaml

```
@@ -1,45 +1,51 @@
 {% set name = "celery-redbeat" %}
-{% set version = "2.0.0" %}
+{% set version = "2.3.2" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: cb8d6941be5df2666dd3188c3029c74e0fa33d50de6eaf7da88e0a5b591a190c
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 7d6e41e4b807fafba5d88b98e3d3863f0df95ac3ed1692d70b134c079b936b8b
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - celery
     - pip
     - python
```

https://github.com/sibson/redbeat/blob/v2.3.2/setup.py#L3
```suggestion
    - python
    - wheel
    - setuptools
```